R/supplement_maf.R

```diff
@@ -16,8 +16,8 @@
 #' my_metadata = GAMBLR.data::gambl_metadata
 #' reddy_meta = dplyr::filter(my_metadata, cohort=="dlbcl_reddy")
 #'
-#' small_maf = GAMBLR.data::sample_dat$maf$grch37 %>%
-#'  dplyr::filter(Tumor_Sample_Barcode %in% reddy_meta$Tumor_Sample_Barcode)
+#' small_maf = dplyr::filter(GAMBLR.data::sample_data$grch37$maf,
```

Comment:
Why not update this example to instead call `get_ssm_by_samples` and give the reddy_meta to `these_samples_metadata` parameter? I think this is a better approach than to fetch SSM by loading the `sample_data` in GAMBLR.data.

Reply:
Vladimir, do you want to address this example? So we can merge this PR as well.

Reply:
Yes. Thank you. I'll do that.

Reply:
Done.